apps/api-e2e/src/integration/app-feature-resolver.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { AppEnv } from '@kin-kinetic/api/app/data-access'
```

Comment:
The `AppEnv` should be imported from `../generated/api-sdk` - just like the other types in this file.